scripts/jenkins-test

```diff
@@ -26,16 +25,24 @@ export SPARK_DRIVER_MEMORY=8g
 
 pushd "$ADAM_TMP_DIR"
 
+
 if [[ $HADOOP_VERSION == "1.0.4" ]]; then
-    wget -q http://d3kbcqa49mib13.cloudfront.net/spark-1.1.0-bin-hadoop1.tgz
-    tar xzvf spark-1.1.0-bin-hadoop1.tgz
-    export SPARK_HOME="${ADAM_TMP_DIR}/spark-1.1.0-bin-hadoop1"
+    HADOOP=hadoop1
+elif [[ $HADOOP_VERSION == "2.6.0" ]]; then
```

Comment:
Maybe `echo $HADOOP_VERSION | egrep '^2.6.'` would be more general, in case, we use e.g. `2.6.3` as the hadoop test version?